spec/inject_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'inject'
+
+describe Array do 
+
+ it 'must respond to jonject' do 
+  expect(subject).to respond_to(:jonject)
+end
+
+  it 'must add up all the values of an array' do 
+  array = [2,3,4,5]
+  expect(array.jonject { |sum,n| sum + n }).to eq 14
+end 
```

Comment:
`end` at 12, 0 is not aligned with `it 'must add up all the values of an array' do ` at 9, 2<br>Trailing whitespace detected.